packages/ui/src/alert-dialog/alert-dialog.tsx

```diff
@@ -0,0 +1,33 @@
+import type { ReactNode } from "react";
+import * as AlertDialog from "@radix-ui/react-alert-dialog";
+
+// ==============================
+// Radix Components
+// https://github.com/radix-ui/primitives/blob/main/packages/react/alert-dialog/src/index.ts
+// ==============================
+export const Root = AlertDialog.Root;
+export const Trigger = AlertDialog.Trigger;
+export const Portal = AlertDialog.Portal;
+export const Overlay = AlertDialog.Overlay;
+export const Content = AlertDialog.Content;
+export const Action = AlertDialog.Action;
+export const Cancel = AlertDialog.Cancel;
+export const Title = AlertDialog.Title;
+export const Description = AlertDialog.Description;
+
+// ==============================
+// CUSTOM OVERRIDES
+// ==============================
+
+// ==============================
+// CUSTOM COMPONENTS
+// ==============================
+
+export const Body = ({ children }: { children: ReactNode }) => (
```

Comment:
What I'd do is export a single component that takes some props such as `description, onCancel, onConfirm` maybe `variant: warn | error`

Reply:
I made #14 to address this

Reply:
I like #14. What would event be better if it exposes these custom components in a way to simply compose them instead of passing them as props.

I find props hard to manage, but I do see the value of custom styled components. If it's only the text we need to pass on to the confirmation button, why wouldn't we simply pass on:
```ts
<Dialog>
  <Dialog.ConfirmationButton>Yes, remove<Dialog.ConfirmationButton/>
</Dialog>
```
?

The advantage I see here is that instead of having to thinkg about how to make props universal, you can create your buttons however you like if you wish to do so but still get the predefined `ConfirmationButton` if you want to reuse something quickly.

Here's thread about [composition](https://twitter.com/mjackson/status/1195495535483817984?s=20&t=b8fh3c6gKyiUrn6zYED_6w) pattern vs props.